distribution/openrct2.d.ts

```diff
@@ -3090,6 +3143,26 @@ declare global {
         readonly item: GuestItemType;
     }
 
+    type StaffAnimation =
+        "walking" |
+        "watchRide" |
+        "wave" |
+        "hanging" |
+        "staffMower" |
+        "staffSweep" |
+        "drowning" |
+        "staffAnswerCall" |
+        "staffAnswerCall2" |
+        "staffCheckboard" |
```

Comment:
```suggestion
        "staffCheckBoard" |
```

Reply:
Oh, nice catch. The capitalisation currently matches the internal enum, but that should be changed for clarity just as well.

Reply:
@AaronVanGeffen You've indeed changed the internal enum now, but not the openrct2.d.ts enum. 🫣